### DISCUSSION TITLE
Whale exploit?

### DISCUSSION THREAD

**Discussion by v4pl** [3kh0/ext-remover#948] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I hear people talk about the wh4le exploit and basically you can open debug window? Anyway here's the link whale.mom

**Comment by AshtonDavies** [comment, marked as answer]
Wh4le is a fake exploit. You should be able to tell by the time it lets you open the "password" window, then it gets silly.

I just don't understand why they bothered to put so much effort into it, buying a domain, programming, explaining it all, and getting contributors just to do all that and to make it look real, because it's not.

Lankeybox, gosh that is the cringiest, funniest thing they could ever put in there, "so cringe, it causes a memory leak." 😂

**Reply by v4pl** [reply]
I haven't tried it yet but I will.

**Reply by AshtonDavies** [reply]
> I haven't tried it yet but I will.

You can try it, but it won't work. The debug buttons do nothing and it's all just inserted HTML from the parent site. The debug window is not actually a Chrome feature.

**Reply by v4pl** [reply]
lol it's basically a scam
